pkg/bloomcompactor/bloomcompactor.go

```diff
@@ -293,6 +293,11 @@ func (c *Compactor) compactUsers(ctx context.Context, logger log.Logger, sc stor
 			return fmt.Errorf("interrupting compaction of tenants: %w", err)
 		}
 
+		// Skip tenant if compaction is not enabled
+		if !c.limits.BloomCompactorEnabled(tenant) {
+			return level.Info(tenantLogger).Log("msg", "compaction disabled for tenant")
```

Comment:
Not sure about returning the return value of `Log()`.
While it works perfectly fine, I think writing the `return` in a separate line looks cleaner.

Reply:
it wasn't suppose to end the loop for this. it has a `continue` on a separate line now.